### DISCUSSION TITLE
Update von 3.67.10.20230114 auf 3.67.10.20230225 schlägt fehl - System bootet nicht

### DISCUSSION THREAD

**Discussion by Troubadix** [OpenCCU/OpenCCU#2241] [🇩🇪 Fragen&Antworten]
### Describe the issue you are experiencing

Update von Raspberrymatic 3.67.10.20230114 auf 3.67.10.20230225 mit Raspberry Pi 4 mit rpi-rf-mod Platine schlägt fehl.

- Die Raspimatic läuft problemlos mit 3.67.10.20230114 und bietet ein Update auf die neue Version 3.67.10.20230225 in der WebUI an.
- Nach Erstellung eines Backups wird das automatische Update durchgeführt, die Raspimatic bootet danach nicht mehr.
- Danach manuelles flashen der 3.67.10.20230225, die Raspimatic bootet. Nach Einspielen des Backups bootet die Raspimatic nicht mehr.

- Auch bei manuellem Update (zip-Datei hier runterladen und in der Raspimatic hochladen) das gleiche Ergebnis.

- Mit der Vorversion 3.67.10.20230114 funktioniert das Backup und lässt sich nach Flashen dieser Version problemlos einspielen.

- Auch Erstellen eines neuen Backups (direkt nach dem Restore, direkt nach Neu flashen, nach Neustart oder auch erst nach einigen Minuten Betrieb) ändert an dem Verhalten nichts.

### Describe the behavior you expected

Update läuft problemlos durch

### Steps to reproduce the issue

1. Backup via WebUI erstellen
2. Update via WebUI
3. System bootet nicht
...

### What is the version this bug report is based on?

3.67.10.20230225

### Which base platform are you running?

rpi4 (RaspberryPi4)

### Which HomeMatic/homematicIP radio module are you using?

RPI-RF-MOD

### Anything in the logs that might be useful for us?

```txt
Da das System nicht bootet komme ich leider auch nicht an die Logs
```

**Comment by jens-maus** [comment]
HDMI Monitor angeschlossen? Was passiert? Neue/Frische SD karte genommen? Was passiert?

P.S: Bitte beachten, das ist kein Supportforum hier, sondern der Bug/Issuetracker. Insofern gehört sowas in den Diskussionsbereich (daher verschoben) und erst in den Bug/Issue Teil wenn das wirklich ein zu bearbeitendes Probelm in RaspberryMatic selbst ist.

**Comment by Troubadix** [comment]
Hallo,

es ist keine SD-Karte sondern eine USB nVME SSD mit 128GB.
Die SMART-Werte dieser sind einwandfrei.

Einen Monitor kann ich nicht anschließen mangels Micro HDMI Kabel.

**Reply by jens-maus** [reply]
> Einen Monitor kann ich nicht anschließen mangels Micro HDMI Kabel.

Adapter kaufen?

**Reply by jens-maus** [reply]
> es ist keine SD-Karte sondern eine USB nVME SSD mit 128GB. Die SMART-Werte dieser sind einwandfrei.

Dann wird es vmtl. an dem USB<>NVMe SSD Adapter liegen. Welcher Typ ist das genau? Spezifikation, Link, etc. Und da bleibt wirklich nur mal nen Monitor verbunden zu bekommen um zu sehen was da passiert.
